modules/ROOT/partials/installing-operator-using-cli-openshift.adoc

```diff
@@ -0,0 +1,56 @@
+// Type: procedure
+[id='installing-operator-using-cli-{context}']
+== Installing the Operator using the CLI
+
+:examples-file: skupper-operator-2.0-ocp-install-examples.zip
+
+The steps in this section show how to use the `kubectl` command-line interface (CLI) to install and deploy the latest version of the {SkupperOperatorName} in a given {kubernetes-flavor} cluster.
+
+.Procedure
+
+. In your web browser, navigate to the *Software Downloads* page for link:https://access.redhat.com/jbossnetwork/restricted/listSoftware.html?downloadType=distributions&product=jboss.amq.interconnect[{skupper-name} releases].
+
+. Ensure that the value of the *Version* drop-down list is set to `{RouterVersion}` and the *Releases* tab is selected.
+
+. Next to *AMQ Interconnect 2.0 Operator Installation and Example Files*, click *Download*.
```

Comment:
Is this going to be named `AMQ Interconnect 2.0 Operator`?

Reply:
AFAIK, the operator will be named 'Skupper Operator', but good question as to what the zip file will be called in customer portal. @mcressman  I'd suggest *AMQ Interconnect 2.0 Installation and Example Files*, wdyt?

Reply:
changed name to Skupper Operator, and we decided not to ship example files at this time